### ISSUE TITLE
Inconsistency between retrieve and list-retrieve

### ISSUE THREAD

**Issue by chad-iris** [iris-edu/rover#99]
Using the test case from #98, it's clear that the `list-retrieve` function does not properly highlight which data `retrive` will actually request.  Likely limited to this particular edge case of a single sample request, where starttime==endtime.

To illustrate, the request:
```
rover retrieve TA_540A_*_* 2010-5-23 2010-5-24
```

The initial request retries up to 3 times before giving up:
```
...
retrieve  DEFAULT: Trying new retrieval attempt 1 of 3.
retrieve  DEFAULT: Downloading TA_540A 2010-143 (N_S 1/1; day 1/1)
retrieve  DEFAULT: Successful retrieval attempt 1 of 3, but no new data downloaded so try again.
retrieve  DEFAULT: Trying new retrieval attempt 2 of 3.
retrieve  DEFAULT: Downloading TA_540A 2010-143 (N_S 1/1; day 1/1)
retrieve  DEFAULT: Successful retrieval attempt 2 of 3, but no new data downloaded so try again.
retrieve  DEFAULT: Trying new retrieval attempt 3 of 3.
retrieve  DEFAULT: Downloading TA_540A 2010-143 (N_S 1/1; day 1/1)
retrieve  DEFAULT: The latest retrieval attempt had no errors.  Stopping at 3 retry attempts.
...
```

If you do `list-retrieve` it reports that _no_ data will be requested and this quite clearly is not true:
```
$ rover list-retrieve TA_540A_*_* 2010-5-23 2010-5-24

  Total: 0 N_S_L_C; 0.00 sec

```

So there is a mis-match between what `retrieve` actually does (properly) in this case and what `list-retrieve` tells the user it would do.

Note: this test case is only possible due to an error in the DMC's extraction system, which will be fixed in the future, so it's a limited time test case.

**Comment by chad-iris** [comment]
A simpler illustration of this issue, starting with an empty local repository, is that this:
```
 rover list-retrieve TA_540A__LHZ 2010-5-23 2010-5-24
```
should return results indicating that the single sample of data returned by the availability service will be requested.

What the availability service returns to ROVER in this case is:
```
#n s     l  c                      earliest                      latest
TA 540A  -- LHZ 2010-05-23T18:15:38.000000Z 2010-05-23T18:15:38.000000Z
```

i.e. one sample.

**Comment by timronan** [comment]
The availabilty service show that there is no data for 

`http://service.iris.edu/iriswsbeta/availability/1/query?format=text&net=TA&sta=540A&loc=*&cha=*&quality=*&starttime=2010-05-23T00:00:00&endtime=2010-05-24T00:00:00&mergequality=false&mergesamplerate=false&mergeoverlap=false&showlastupdate=false&excludetoolarge=true&includerestricted=false&nodata=404`

maybe I am missing something but we may need to come up with a different test case.

**Comment by chad-iris** [comment]
@timronan That URL is for the _beta_ service.

For `rover list-retrieve TA_540A__LHZ 2010-5-23 2010-5-24`

Here is the equivalent availability query:

http://service.iris.edu/irisws/availability/1/query?format=text&net=TA&sta=540A&loc=--&cha=LHZ&starttime=2010-05-23T00:00:00.000000&endtime=2010-05-24T00:00:00.000000&mergequality=true&mergesamplerate=true&nodata=404

**Comment by timronan** [comment]
Closed by pull request #108.
